--- gutenprint-5.2.14/configure.ac.orig	2019-03-23 21:09:21.792451636 +0100
+++ gutenprint-5.2.14/configure.ac	2019-03-23 21:10:02.135566412 +0100
@@ -353,24 +353,7 @@
     dlopen) [WITH_MODULES]="dlopen";  AC_MSG_RESULT([dlopen]) ;;
     static) [WITH_MODULES]="static";  AC_MSG_RESULT([static]) ;;
     *) AC_MSG_ERROR([${withval}: not a valid module loader]) ;;
-                      ],[
-    # if static libraries are enabled, modules must be static
-    if test x$enable_static = xyes; then
-      [WITH_MODULES]="static"
-    else
-      [WITH_MODULES]="yes"]
-    fi)
-
-# Abort if a broken configuration was required
-if test x$enable_static = xyes; then
-  if test $WITH_MODULES = no; then
-    :
-  elif test $WITH_MODULES = static; then
-    :
-  else
-    AC_MSG_ERROR([--disable-static must be used with --with-modules])
-  fi
-fi
+                      ],[WITH_MODULES="yes"])
 
 STP_ARG_ENABLE([debug],
                [turn on debugging in build],
