--- gutenprint-5.3.4/configure.ac.orig	2020-06-05 01:28:41.000000000 +0200
+++ gutenprint-5.3.4/configure.ac	2023-08-28 21:38:43.451204802 +0200
@@ -753,17 +753,6 @@ if test x$ac_compiler_gnu = "xyes"; then
   if test x${USE_MAINTAINER_MODE} = xyes ; then
     STP_ADD_COMPILER_ARGS([ -pedantic -Waggregate-return -Wcast-qual -Wshadow -Wredundant-decls],, [GNUCFLAGS])
   fi
-  if test x$ENABLE_DEBUG = xyes ; then
-    STP_ADD_COMPILER_ARG([-g])
-  else
-    STP_ADD_FIRST_COMPILER_ARG([-O3 -O2 -O1 -O])
-  fi
-else
-  if test x$ENABLE_DEBUG = xyes ; then
-    STP_ADD_COMPILER_ARG([-g])
-  else
-    STP_ADD_FIRST_COMPILER_ARG([-O])
-  fi
 fi
 if test x$ENABLE_PROFILE = xyes ; then
   STP_ADD_COMPILER_ARG([-pg])
@@ -1019,13 +1008,6 @@ AM_CONDITIONAL(BUILD_MODULES, test x$MOD
 
 AM_CONDITIONAL(BUILD_GENPPD_STATIC, test x"$STATIC_GENPPD" = xyes)
 
-dnl Define LTLIBOBJS
-AC_CONFIG_COMMANDS_PRE(
-[LTLIBOBJS=`echo "$LIB@&t@OBJS" | sed 's/\.o/.lo/g'`
-AC_SUBST(LTLIBOBJS)
-LTALLOCA=`echo "$ALLOCA" | sed 's/\.o/.lo/g'`
-AC_SUBST(LTALLOCA)])
-
 dnl Definitions
 AC_DEFINE_UNQUOTED(CUPS_DATADIR, "${cups_conf_datadir}", [CUPS data directory.])
 
