--- gutenprint-5.2.11/configure.ac.orig
+++ gutenprint-5.2.11/configure.ac
@@ -371,9 +371,9 @@ if test "$BUILD_FOOMATIC_3" = "yes" ; then
    BUILD_FOOMATIC=yes
 fi
 
-AC_PATH_PROG(IJS_CONFIG, ijs-config)
+PKG_CHECK_MODULES([IJS], [ijs], [AC_DEFINE([HAVE_IJS], [1], [Use IJS]) IJS_OK=yes], [IJS_OK=no])
 
-if test -z "${IJS_CONFIG}"  ; then
+if test "x${IJS_OK}" = "xyes"; then
   BUILD_GHOSTSCRIPT="no"
 else
   BUILD_GHOSTSCRIPT="yes"
@@ -602,11 +602,9 @@ fi
 
 dnl ijs checks
 if test x${BUILD_GHOSTSCRIPT} = xyes ; then
-  if test -z "$IJS_CONFIG" ; then
+  if test "$IJS_OK" != yes; then
     AC_MSG_ERROR(Cannot find ijs-config; please ensure Ghostscript 6.53 or above is installed);
   fi
-  IJS_CFLAGS=`$IJS_CONFIG --cflags`
-  IJS_LIBS=`$IJS_CONFIG --libs`
 fi
 
 GUTENPRINT_RELEASE_VERSION=${GUTENPRINT_MAJOR_VERSION}.${GUTENPRINT_MINOR_VERSION}
